clang/lib/Sema/SemaOverload.cpp

```diff
@@ -14474,6 +14474,23 @@ ExprResult Sema::CreateOverloadedBinOp(SourceLocation OpLoc,
                                        CurFPFeatureOverrides());
   }
 
+  // If this is the .* operator, which is not overloadable, just
+  // create a built-in binary operator.
+  if (Opc == BO_PtrMemD) {
+    auto CheckPlaceholder = [&](Expr *&Arg) {
+      ExprResult Res = CheckPlaceholderExpr(Arg);
+      if (!Res.isInvalid())
+        Arg = Res.get();
+      return Res.isInvalid();
```

Comment:
Do we perhaps want to use `!Res.isUsable()` because `CreateBuiltinBinOp()` expects the `Expr *`s to be nonnull?

Reply:
I was wondering what the difference between those two was, so thanks for pointing that out. We definitely want that one then.